packages/deployment/Dockerfile.sdk

```
@@ -1,6 +1,6 @@
 ###########################
 # The golang build container
-FROM golang:1.17-buster as cosmos-go
+FROM golang:1.18 as cosmos-go
```

This turns out to be too big of a jump.  My PR also upgrades `buster` to `bullseye`, which is probably not what we want.  Instead:
```suggestion
FROM golang:1.18-buster as cosmos-go
```